airbyte-integrations/connectors/source-hubspot/source_hubspot/helpers.py

```diff
@@ -110,7 +105,11 @@ class APIv1Property(IURLPropertyRepresentation):
     def as_url_param(self):
         return {"property": self.properties}
 
-    def as_url_param_with_history(self) -> str:
+
+class APIv1PropertiesWithHistory(IURLPropertyRepresentation):
```

Comment:
What the differences is between `APIv1PropertiesWithHistory` and `APIv2PropertiesWithHistory`?

Reply:
Indeed there is no reason to have two versions for now.
Fixed